Added a comment for the sphinx-build -j option (#12837)
sphinx-doc/sphinx: doc/man/sphinx-build.rst

.. option:: -j N, --jobs N

   Distribute the build over *N* processes in parallel, to make building on

>    multiprocessor machines more effective. 
>    This feature only works on systems supporting "fork". Windows is not supported.
>    Note that not all parts and not all builders of Sphinx can be parallelized.
>    If ``auto`` argument is given,

Reviewer: ```suggestion
   multiprocessor machines more effective.
   This feature only works on systems supporting "fork". Windows is not supported.
   Note that not all parts and not all builders of Sphinx can be parallelized.
   If ``auto`` argument is given,
```

.. option:: -j N, --jobs N

   Distribute the build over *N* processes in parallel, to make building on

>    multiprocessor machines more effective. This feature only works on systems
>    supporting "fork", e.g., linux. Windows is not supported. Also not all parts
>    and not all builders of Sphinx can be parallelized.  If ``auto`` argument is given,

Reviewer: ```suggestion
   multiprocessor machines more effective. 
   This feature only works on systems supporting "fork". Windows is not supported.
   Note that not all parts and not all builders of Sphinx can be parallelized.
   If ``auto`` argument is given,
```
